src/proto/single/ent/mod.rs

```diff
@@ -126,8 +126,8 @@ mod test {
 
     #[test]
     fn test_same_purpose_setters_applied_simultaneously() {
-        let expires_at0 = Utc::now() + chrono::Duration::seconds(300);
-        let expires_at1 = Utc::now() + chrono::Duration::seconds(300);
+        let expires_at0 = Utc::now() + chrono::Duration::try_seconds(300).unwrap();
+        let expires_at1 = Utc::now() + chrono::Duration::try_seconds(300).unwrap();
```

Comment:
This is interesting: I made this change to satisfy the `ubuntu/beta/update` job, which made the `minimal-versions` watch-dog job fail :)

Reply:
Those are actually changes in `chrono` rather than `time`. I thought `TimeDelta` is a construct from `time` crate, but it is not. This is a new name for `Duration`, which makes perfect sense since it supports negative values, delta - is a better naming.

Those `try_<unit>` methods have been [added](https://github.com/chronotope/chrono/commit/a2820c4f7db20a77d69bdb62d822394c52db0aa8) in September 2023 and [released](https://github.com/chronotope/chrono/releases/tag/v0.4.32) in chrono@0.4.32